apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/SpannerStressTests.cs

```diff
@@ -131,6 +132,10 @@ public SpannerStressTests(TestDatabaseFixture testFixture, ITestOutputHelper out
 
         private async Task RunStress(Func<Stopwatch, Task<TimeSpan>> writeFunc)
         {
+            // Clear current session pool to eliminate the chance of a previous
```

Comment:
its part of the change -- very indirectly.
What's going on is that because clients are now indexed on their full set of options (which is by design), some of the tests that set AllowZeroTimeouts are creating pools that do not otherwise get used -- causing a pretty significant difference in their sizes.
this does a full reset before the stress test, which allows all other tests to use different options without messing up this test.